### PR TITLE
Bugfixes, quality of life changes

### PR DESCRIPTION
**Pull request by 3141592654**
@mhrmm Of which the most noticeable is I removed which Pokemon was on the field from the state vector. Among other things, this cuts the memory used by the saves by about four-fifths.

### CODE REVIEW

File: bookkeeper.py [hunk 1]
```diff
@@ -63,40 +63,57 @@ def report_observation(observation):
...
-                    index += NUM_STATUS_CONDITIONS
+                    index += TEAM_SIZE * NUM_STATUS_CONDITIONS
                 elif index < OFFSET_STAT_BOOSTS:
-                    index -= NUM_STATUS_CONDITIONS
+                    index -= TEAM_SIZE * NUM_STATUS_CONDITIONS
                 elif index < OFFSET_STAT_BOOSTS + NUM_STAT_BOOSTS:
                     index += NUM_STAT_BOOSTS
                 elif index < OFFSET_WEATHER:
                     index -= NUM_STAT_BOOSTS
+                elif index < OFFSET_TERRAIN:
```
Comment (mhrmm) [inline bookkeeper.py:94]: Add unit tests at some point later?

File: pkmn.py [hunk 2]
```diff
@@ -293,8 +263,9 @@ def opponent_choose_action(x, bookkeeper, action_space):
             # with values greater than 32.
             pvec[i] -= pvec_max - 32
     pvec = np.exp(pvec)
-    for i in range(len(OPPONENT_POSSIBLE_ACTIONS)):
-        pvec[i] *= OPPONENT_POSSIBLE_ACTIONS[i] in action_space
+    if (len(sys.argv) == 2 or debug):
```
Comment (mhrmm) [inline pkmn.py]: Maybe set debug = True if len(sys.argv) == 2, just to make a bit more robust.

**Review by mhrmm — COMMENTED** [review]
LGTM

File: bookkeeper.py (hunk 1 above)
Comment (3141592654) [reply bookkeeper.py:94]: Decided unit tests would require too much rewriting, added utility file with similar functionality instead

File: pkmn.py (hunk 2 above)
Comment (3141592654) [reply pkmn.py]: Done

**Pull request merged**